website/content/docs/deprecation/index.mdx

```diff
@@ -1,44 +1,126 @@
+
+@include 'deprecation/ruby-client-library.mdx'
+
+@include 'deprecation/active-directory-secrets-engine.mdx'
+
+@include 'deprecation/centrify-auth-method.mdx'
+
+</Tab>
+<Tab heading="PENDING REMOVAL">
+
+@include 'deprecation/vault-agent-api-proxy.mdx'
+
+</Tab>
+<Tab heading="REMOVED">
+
+@include 'deprecation/duplicative-docker-images.mdx'
+
+@include 'deprecation/aws-field-change.mdx'
+
+@include 'deprecation/azure-password-policy.mdx'
+
+</Tab>
+</Tabs>
+
+## Deprecation phases
+
+The lifecycle of a Vault feature or plugin includes 4 phases:
+
+- **supported** - generally available (GA), functioning as expected, and under
+  active maintenance
+- **deprecated** - marked for removal in a future release
+- **pending removal** - support ended or replaced by another feature
+- **removed** - end of lifecycle
+
+### Deprecated ((#deprecated))
+
+"Deprecated" is the first phase of the deprecation process and indicates that
+the feature is marked for removal in a future release. When you upgrade Vault,
+newly deprecated features will begin alerting that the feature is deprecated:
+
+- Built-in authentication and secrets plugins log `Warn`-level messages on
+  unseal.
+- All deprecated features log `Warn`-level messages.
+- All `POST`, `GET`, and `LIST` endpoints associated with the feature return
+  warnings in response data.
+
+Built-in Vault authentication and secrets plugins also expose their deprecation
+status through the Vault CLI and Vault API.
+
+CLI command                                                                  | API endpoint
+---------------------------------------------------------------------------- | --------------
+N/A                                                                          | [`/sys/plugins/catalog`](/vault/api-docs/system/plugins-catalog)
+[`vault plugin info auth <PLUGIN_NAME>`](/vault/docs/commands/plugin/info)   | [`/sys/plugins/catalog/auth/:name`](/vault/api-docs/system/plugins-catalog#list-plugins-1)
+[`vault plugin info secret <PLUGIN_NAME>`](/vault/docs/commands/plugin/info) | [`/sys/plugins/catalog/secret/:name`](/vault/api-docs/system/plugins-catalog#list-plugins-1)
+
+### Pending removal
+
+"Pending removal" is the second phase of the deprecation process and indicates
+that the feature behavior is fundamentally altered in the following ways:
+
+- Built-in authentication and secrets plugins log `Error`-level messages and
+  cause an immediate shutdown of the Vault core.
+- All features pending removal fail and log `Error`-level messages.
+- All CLI commands and API endpoints associated with the feature fail and return
+  errors.
+
+<Warning title="Use with caution">
+
+  In critical situations, you can override the pending removal behavior with the
+  [`VAULT_ALLOW_PENDING_REMOVAL_MOUNTS`](/vault/docs/commands/server#vault_allow_pending_removal_mounts)
+  environment variable, which forces Vault to treat features that are pending
+  removal as if they were still only deprecated.
```

Comment:
```suggestion
  In critical situations, you may be able to override the pending removal behavior with the
  [`VAULT_ALLOW_PENDING_REMOVAL_MOUNTS`](/vault/docs/commands/server#vault_allow_pending_removal_mounts)
  environment variable, which forces Vault to treat some features that are pending
  removal as if they were still only deprecated.
```

This environment variable only applies to some features that are deprecated. Roughly, secrets engines and auth methods that are built in to the binary instead of external plugins. I've made a tentative suggestion to try to make it clearer that this env var won't work for everything, which I think we should make more clear, but I'm not saying my suggestion has to be the verbiage.